File: 217-contains-duplicate/217-contains-duplicate.cpp
class Solution {
public:
    bool containsDuplicate(vector<int>& nums) {
        unordered_map<int,int>mp;
        for(int i : nums){
            mp[i]++;
            if(mp[i]>1)
                return true;
        }
        return false;
    }
};

File: 217-contains-duplicate/217-contains-duplicate.java
class Solution {
    public boolean containsDuplicate(int[] nums) {
        HashMap<Integer, Integer> mp = new HashMap<>();
        for(int i=0;i< nums.length;i++){
            if(mp.containsKey(nums[i]))
                return true;
            else 
                mp.put(nums[i],1);
        }
        return false;
        
    }
}